src/dropdown/dropdown.js

```diff
@@ -15,43 +16,69 @@ import DropdownPanel from './dropdownpanel.js';
 import DropdownPanelView from './dropdownpanelview.js';
 
 /**
- * The basic dropdown controller class.
+ * The Dropdown controller class.
+ *
+ *		const model = new Model( {
+ *			label: 'Dropdown',
+ *			isEnabled: true
+ *		} );
+ *
+ *		// An instance of Dropdown.
+ *		new Dropdown( model, new DropdownView() );
+ *
+ * See {@link ui.dropdown.DropdownView}.
  *
  * @memberOf ui.dropdown
  * @extends ui.Controller
  */
 export default class Dropdown extends Controller {
 	/**
-	 * Creates a Dropdown instance.
+	 * Creates an instance of {@link ui.dropdown.Dropdown} class.
 	 *
-	 * @param {utils.Observable} model
-	 * @param {ui.View} view
+	 * @param {ui.dropdown.DropdownModel} model Model of this Dropdown.
+	 * @param {ui.View} view View of this Dropdown.
 	 */
 	constructor( model, view ) {
 		super( model, view );
 
 		this.collections.add( new ControllerCollection( 'main' ) );
 
 		/**
+		 * An internal Model interface shared between sub–components of the Dropdown.
+		 *
+		 * @member {ui.dropdown.DropdownSharedModel} ui.dropdown.Dropdown#sharedModel
+		 */
+		this.sharedModel = new Model( {
```

Comment:
It solves https://github.com/ckeditor/ckeditor5-ui/issues/31. I don't like it but I didn't see any other option at the moment of coding. Let's discuss it.